assets/js/blocks/reviews/reviews-by-product/edit.js

```diff
@@ -49,14 +49,13 @@ const ReviewsByProductEditor = ( {
 				countLabel={ sprintf(
 					/* translators: %d is the review count. */
 					_n(
-						'%d Review',
-						'%d Reviews',
+						'%d review',
+						'%d reviews',
```

Comment:
We didn't have a consistent casing style for strings shown in the count bubble. In 981fa518f5697eb51e623b16b755363636636b77 I made all of them to use sentence casing.